.github/workflows/build.yml

```diff
@@ -0,0 +1,66 @@
+name: Publish Fireshape's Docker images
+
+on:
+  # Push to master or PR
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Currently:
```
on:
  # Push to master or PR
  push:
    branches:
      - master
  pull_request:
  # build every week to stay up-to-date with firedrake
  schedule:
    - cron: '0 0 * * 0'
```

Will build docker images on every PR, I think you need `if: ${{ github.ref == 'refs/heads/master' }}` before `runs on:` in each job to only build docker images on "push to master"

Reply:
Thanks for pointing this out. Fixed.